.gitignore

```
@@ -70,17 +70,20 @@ WordPress/Images.xcassets/AppIcon-Internal.appiconset
 /vendor/
 
 # Fastlane
-Scripts/fastlane/Preview.html
-Scripts/fastlane/report.xml
-Scripts/fastlane/README.md
-Scripts/Preview.html
-Scripts/fastlane/test_output/
-Scripts/fastlane/google_cloud_keys.json
-Scripts/fastlane/promo_screenshots
-/Scripts/fastlane/metadata/review_information
-/Scripts/default.profraw
+fastlane/Preview.html
+fastlane/report.xml
+fastlane/README.md
+Preview.html
+fastlane/test_output/
+fastlane/google_cloud_keys.json
+fastlane/promo_screenshots
+fastlane/metadata/review_information
+default.profraw
 derived-data/
 
+# CI Artifacts Location
+/Artifacts
```

```suggestion
Artifacts
```

And actually similar for other existing entries below like `/WordPress/…` lines and such

---

Cleaned up in bf2ccf60b8bbed5ebe02fe1cb1d8dd5558db2e5e